python/python/tests/test_fts.py

```diff
@@ -142,12 +142,77 @@ def test_search_fts(table, use_tantivy):
     assert len(results) == 5
 
 
+def test_search_fts_specify_column(table):
+    table.create_fts_index("text", use_tantivy=False)
+    table.create_fts_index("text2", use_tantivy=False)
+
+    results = table.search("puppy", fts_columns="text").limit(5).to_list()
+    assert len(results) == 5
+
+    results = table.search("puppy", fts_columns="text2").limit(5).to_list()
+    assert len(results) == 5
```

Comment:
Could we change this so it uses two different text columns? Right now `text` and `text2` are identical. Because of this, this test wouldn't be able to detect whether this parameter is ignored and just the first text column available is used. The results of this query should be different, and we should be able to assert they are different.

Reply:
Fixed